block/bench/Block.hs

```diff
@@ -28,6 +28,7 @@ import           Pos.Core.Common (CoinPortion, SharedSeed (..))
 import           Pos.Core.ProtocolConstants (ProtocolConstants (..))
 import           Pos.Core.Genesis
 import           Pos.Crypto (ProtocolMagic (..))
+import           Test.Pos.Block.Arbitrary.Generate (generateMainBlock)
```

Comment:
And there should be a line above this. It's good to separate the test modules from the main modules with extra whitespace.